optimade/server/config.py

```diff
@@ -115,8 +116,8 @@ def _DEFAULTS(field: str) -> Any:
                 "prefix": "_exmpl_",
                 "name": "Example provider",
                 "description": "Provider used for examples, not to be assigned to a real database",
-                "homepage": "http://example.com",
-                "index_base_url": "http://example.com/optimade/index",
+                "homepage": "https://example.com",
+                "index_base_url": "https://example.com/index/optimade",
```

Comment:
ah, now I get it.

I would actually suggest to use the same base URL for the index DB (`https://example.com/optimade`).
Also, I would suggest to remove the index DB from the default `provider` response - again, since most people won't run an index DB.

Reply:
No. The endpoint (or OPTiMaDe base URL) should end with either `/optimade`, `/optimade/v0.10` or `/optimade/v0.10.0` with the current version.
This was an old mistake I did, which has been fixed everywhere else in the code, but was missed here.

Reply:
> Also, I would suggest to remove the index DB from the default `provider` response - again, since most people won't run an index DB.

Well - if they want to provide a base URL, from where their databases/implementation can be discovered, they would have to, I believe?

Reply:
> Well - if they want to provide a base URL, from where their databases/implementation can be discovered, they would have to, I believe?

Ah, I think I misunderstood: this is not the url of the index metadatabase, but it is supposed to be the (non-version-specific) base URL.
In that case I agree that it makes sense. 
I just remember seeing a couple of port 5001 (instead of 5000) in the responses and thought it was for the meta database. I'll have to check again later.

Reply:
No, I meant. If people want to have their databases be discoverable, they would need to run an index database. The port 5001 URLs are indeed to the index meta-database as it is set up in this repository by default.
But sure, `index_base_url` is an OPTIONAL field, so we could leave it out, it was merely to provide an example of what it may look like.